packages/cms/src/utils/varSwapRecursive.js

```diff
@@ -50,7 +50,8 @@ const varSwapRecursive = (sourceObj, formatterFunctions, variables, query = {},
         obj[skey] = obj[skey].filter(allowed).map(o => varSwapRecursive(o, formatterFunctions, variables, query, selectors));
       }
       // If this property is an object, recursively do another swap
-      else if (typeof obj[skey] === "object" && obj[skey] !== null) {
+      // For some reason, postgres "DATE" props come through as objects. Exclude them from this object swap.
```

Comment:
TYL: In JavaScript, `Date` variables are considered "objects", along with functions and arrays 🙄